### PR TITLE
Stories/ecer 3671

### PR DESCRIPTION
**Pull request by farzadnadiri**
ECER-3671: ECE-EW supporting API

GET /api/certifications/files/[summaryid?]
GET /api/certifications/file/download/[summaryid]

## Checklist
- [ ] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)
Include screenshots to demonstrate the changes visually.

## Additional Comments (optional)
Any additional context or information that might be helpful for the reviewer.

### CODE REVIEW

File: src/ECER.Tests/ECER.Tests.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
     <NoWarn>CA1001;1701;1702;CA1707</NoWarn>
+    <UserSecretsId>2435d405-b699-4b67-bfde-d7d73e7c2c6c</UserSecretsId>
```
Comment (ytqsl) [inline src/ECER.Tests/ECER.Tests.csproj:8]: need to revert this change

**Review by ytqsl — CHANGES_REQUESTED** [review]

File: src/ECER.Tests/ECER.Tests.csproj (hunk 1 above)
Comment (farzadnadiri) [reply src/ECER.Tests/ECER.Tests.csproj]: OOps, done!

**Review by ytqsl — APPROVED** [review]
